build.sbt

```diff
@@ -76,11 +115,24 @@ lazy val frontEnd = project("front-end")
     resolvers += Resolver.bintrayRepo("typesafe", "maven-releases"),
     ReactJsKeys.sourceMapInline := true,
     // ConductR settings
+    BundleKeys.bundleConfVersion := BundleConfVersions.V_1_2_0,
     BundleKeys.nrOfCpus := 1.0,
     BundleKeys.memory := 64.MiB,
     BundleKeys.diskSpace := 35.MB,
-    BundleKeys.endpoints := Map("web" -> Endpoint("http", services = Set(URI("http://:9000")))),
-    javaOptions in Bundle ++= Seq("-Dhttp.address=$WEB_BIND_IP", "-Dhttp.port=$WEB_BIND_PORT")
+//    BundleKeys.endpoints := Map("web" -> Endpoint("http", services = Set(URI("http://:9000")))),
+    BundleKeys.endpoints := Map(
+      "chirperweb" -> Endpoint("http", 0, "chirperweb",
+        RequestAcl(
+          Http(
+            "^/chirper".r
+          ))
+      )
+    ),
+    javaOptions in Bundle ++= Seq(
+      "-Dapplication.context=/chirper/",
```

Comment:
You wouldn't have to given generated acls. Nice to see how it is done though. :-)